docs/Web-Interface.md

```diff
@@ -98,4 +98,28 @@ If the web browser and the Transmission daemon are on the machine you can use ht
 | 6                 | Progress bar showing the current progress in downloading the torrent            |
 | 7                 | Pause selected torrent                                                          |
 
+### Torrent inspector ###
+
+
+| Number On Picture | Description                                                                                                                                |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| 1                 | Name of torrent                                                                                                                            | 
+| 2                 | In current case: Transmission have 3.65 GB of file data from the total volume of downloaded files, which is 100%                           |
```

Comment:
This should probably read „Transmission has“. The app is singular.

Reply:
Ok, i fix it in next PR!